judge/models/problem.py

```diff
@@ -535,3 +558,38 @@ class Meta:
         )
         verbose_name = _('solution')
         verbose_name_plural = _('solutions')
+
+
+class ProblemPointsVote(models.Model):
+    points = models.IntegerField(
+        verbose_name=_('proposed points'),
+        help_text=_('the amount of points the voter thinks this problem deserves.'),
+        validators=[
+            MinValueValidator(settings.DMOJ_PROBLEM_MIN_USER_POINTS_VOTE),
+            MaxValueValidator(settings.DMOJ_PROBLEM_MAX_USER_POINTS_VOTE),
+        ],
+    )
+
+    voter = models.ForeignKey(Profile, related_name='problem_points_votes', on_delete=CASCADE)
+    problem = models.ForeignKey(Problem, related_name='problem_points_votes', on_delete=CASCADE)
+    vote_time = models.DateTimeField(
+        verbose_name=_('vote time'),
+        help_text=_('the time this vote was cast'),
+        auto_now_add=True,
+        blank=True,
```

Comment:
What's the point of having `blank=True`?

Reply:
From the django docs,
> As currently implemented, setting auto_now or auto_now_add to True will cause the field to have editable=False and blank=True set.

I wasn't aware of this and it's redundant, so I'll remove it.